### DISCUSSION TITLE
Support for Azure Linux 3.0 (AKA Mariner)

### DISCUSSION THREAD

**Discussion by eric-desrochers** [aquasecurity/trivy#6662] [Ideas]
### Description

What would you like to be added:
(With 3.0, Mariner has been fully rebranded to Azure Linux.)
Add support for Azure Linux 3.0.

(Note: Mariner 2.0 must continue to be supported as a major stable release.)

Why is this needed:
CVE detection.

Additional context:
`/etc/os-release`
ID=**azurelinux**

OVAL file:
https://github.com/microsoft/AzureLinuxVulnerabilityData/

### Target

None

### Scanner

Vulnerability

**Comment by knqyf263** [comment]
Is Azure Linux 3.0 already released?

**Reply by eric-desrochers** [reply]
Not yet officially GA but I can share early images for you to develop the support.

**Reply by knqyf263** [reply]
Created https://github.com/aquasecurity/trivy/issues/6673.
We look forward to contributions from Microsoft.
